test.sh

```diff
@@ -8,10 +8,6 @@ if [ -e $p ]; then
     \rm -rf $p || exit 1
 fi
 
-# RTDB now requires this environment variable to be set
```

Comment:
Explicitly unset for testing? Or maybe also consider to throw a more explicit error if set. I had some funny failures when the env was still present.